inference-engine/src/transformations/src/transformations/common_optimizations/moc_transformations.cpp

```diff
@@ -79,6 +81,15 @@ bool ngraph::pass::MOCTransformations::run_on_function(std::shared_ptr<ngraph::F
     if (!m_use_shapes) {
         manager.register_pass<ngraph::pass::DisableShapeOfConstantFolding>();
     }
+    // RemoveConcatZeroDimInput and RemoveMultiSubGraphOpDanglingParams
+    // should be performed before first ConstantFolding call.
+    // The passes can deteach graph branches where zero dimesion is calculated.
+    // Zero dimensions in shape causes creation empty tensors, which are incorrect during CT.
```

Comment:
Did you mean that?
```suggestion
    // Zero dimensions in shape causes creation empty tensors, which are incorrect during CF.
```

Reply:
@jane-intel  Good catch. If no more requests can I commit it separately?

Reply:
done